app/assets/javascripts/pdfjs_viewer/viewer.js

```diff
@@ -7133,10 +7133,18 @@ var pdfjsWebLibs;
    var activeService = null;
    function renderPage(activeServiceOnEntry, pdfDocument, pageNumber, size) {
     var scratchCanvas = activeService.scratchCanvas;
+
+    var MAX_CANVAS_AREA = 268435456; // Per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/canvas#Maximum_canvas_size
     var PRINT_RESOLUTION = 600;
-    var PRINT_UNITS = PRINT_RESOLUTION / 72.0;
-    scratchCanvas.width = Math.floor(size.width * PRINT_UNITS);
-    scratchCanvas.height = Math.floor(size.height * PRINT_UNITS);
+    do {
+      var PRINT_UNITS = PRINT_RESOLUTION / 72.0;
```

Comment:
It's confusing that this var is ALL_CAPS because it no longer is acting as much like a constant.